index.html

```diff
@@ -1,10 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="author" content="Kaboha Jean Mark">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Basic HTML Boilerplate</title>
-        <link rel="stylesheet" href="css/styles.css">
-    </head>
-</html>
\ No newline at end of file
+<head>
+    <meta charset="UTF-8" />
+    <meta name="author" content="Kaboha Jean Mark">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pele Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body id="main">
+  <header>
+    <div class="content-wrap">
+      <h1 id="title">Pele</h1>
+      <p>The man who made football known as the beautiful game</p>
+     </div>
+  </header>
+  
+  <main>
+    <div id="img-div" class="content-wrap">
+      <figure>
+        <img id="image" src="https://images.pexels.com/photos/46798/the-ball-stadion-football-the-pitch-46798.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940">
+        <figcaption id="img-caption">A football in the middle of a stadium</figcaption>
+       </figure>
+    </div>
+    <div id="tribute-info">
```

Comment:
```suggestion
    <section id="tribute-info">
```
Perhaps it would be nice to have all of this content in a section tag for more semantic HTML too and improving the SEO like in the upper section.